[en] sync ghippo release-notes (#4505)
DaoCloud/DaoCloud-docs: docs/en/docs/ghippo/intro/release-notes.md

### Features

> - **Added** Platform settings: button for background video on the login page

Reviewer: ```suggestion
- **Added** an option in Platform Settings to show/hide the background video on the login page
```
原文还需要改改，不像一句话

> - **Added** Personal center: features of view/add/update/delete SSH certificate information

Reviewer: ```suggestion
- **Added** features to view/add/update/delete SSH certificate information in Personal Center
```

- **Added** features of creating user and changing password:  encryption for username and password parameters when
  passed to API
- **Added** MySQL MGR mode

### Optimization

- **Optimized** Keycloak component to version 22.0.4

### Fixes

> - **Fixed** issue with GPU count statistics after licensing by GPU

Reviewer: ```suggestion
- **Fixed** an issue with GPU count statistics after licensing by GPU
```
有 an 更好一点，可以全文统改

## 2024-4-30

### v0.26.0

### Features

- **Added** support for license by GPU
- **Added** a summary link in Audit Logs

### Optimization

- **Optimized** unit tests for the Audit Logs SDK

### Fixes

- **Fixed** an issue with the user-agent field in Audit Logs
- **Fixed** an issue where Audit Logs login records often show one failure followed by one success
- **Fixed** an issue in __Operations Management__ where validation prompts were not removed after clicking cancel on 
  billing configuration changes
- **Fixed** an issue in __Operations Management__ with GPU billing duration calculation in pod reports

## 2024-4-1

### v0.25.1

### Fixes

> - **Fixed** an issue with workspace resource group binding namespace resource quota checks

Reviewer: ```suggestion
- **Fixed** an issue with resource quota check while workspace resource group is binding to a namespace
```

- **Fixed** issue with workspace resource group binding namespace resource quota checks

## 2024-3-31

### v0.25.0

### Features

- **Added** a feature where top-level navigation bar is displayed based on permissions
- **Added** a feature where Workspace/Folder Editor does not support changing Workspace/Folder and names
- **Added** Ghippo support for ARM offline installation packages
- **Added** a method provided by SDK to find Alias using Workspace ID GetWorkspaceById()
- **Added** __Operations Management__ report management -> pod report: GPU statistics

> - **Added** __Operations Management__ billing and accounting -> pod billing: GPU billing

Reviewer: ```suggestion
- **Added** __Operations Management__ -> Accounting and Billing -> Pod Billing: GPU billing
```
所有 UI 词条，要跟 UI 一致，包括大小写

### Fixes

- **Fixed** issue with front-end CSS errors on global management login
- **Fixed** issue where the Refresh Token API in the observability interface could not update the Token

## 2024-1-31

### v0.24.0

#### Features

- **Added** resource quotas limiting GPU
- **Added** support for DCE5 logged-in users to access Insight's Grafana and other components

#### Optimization

- **Optimized** rate limiting with single flight mechanism when triggering FoldersAuthz CR
- **Optimized** filtering duplicate authorizations
- **Optimized** the condition where Workspace authorization information is not stored in FoldersAuthz CR
- **Optimized** a full update of FoldersAuthz CR upon ghippo-controller-manager restart

#### Fixes

- **Fixed** issue where a large number of authorization requests caused FoldersAuthz CR updates to overwhelm k8s

## 2023-12-29

### v0.23.0

#### Features

> - **Added** a function to automatically log out from the identity provider page when logging out of DCE 5.0

Reviewer: ```suggestion
- **Added** a feature to automatically log out from the identity provider page when logging out of DCE 5.0
```
基本都是 feature

- **Added** support for usernames with underscores and other special characters
- **Added** removal of the restriction on authorizing usernames with special characters
- **Added** user isolation mode between Folders
- **Added** support for authorizing multiple roles to Folder/Workspace users

#### Optimization

- **Optimized** initial username and password (admin/changeme) to be stored in Secret instead of ConfigMap

#### Fixes

- **Fixed** issue with the redirection of offline environment documentation site
- **Fixed** issue with installer version 0.13 upgrade failure
- **Fixed** issue with workspace large-scale test performance

## 2023-12-05

### v0.22.1

#### Fixes

- **Fixed** issue with incomplete display of middleware in the navigation bar

## 2023-11-30

### v0.22.0

#### Features

- **Added** auto-expanding active navigation bar
- **Added** SDK support for usernames with underscores
- **Added** support for DCE 4.0 to DCE 5.0 migration validation
- **Added** __Operations Management__ precise search in list

#### Optimization

> - **Optimized** platform - product version submodules kcoral, dowl, kcollie, virtnest support Chinese

Reviewer: original is better

#### Fixes

- **Fixed** issue where LDAP users synchronized over could not be added to user groups
- **Fixed** issue in Operations Management where workspace reports were not fully displayed

## 2023-11-01

### v0.21.0

#### Features

> - **Added** a feature of integrating multiple AD/LDAP

Reviewer: ```suggestion
- **Added** a feature of integrating multiple ADs/LDAPs
```

> - **Added** a feature of integrating multiple OIDC

Reviewer: ```suggestion
- **Added** a feature of integrating multiple OIDCs
```

## 2023-07-28

### v0.19.0

#### Features

- **Added** support for integrating project group IDP login without clicking
- **Added** resource group binding to mesh / mesh-namespace
- **Added** tips for custom role permission points
- **Added** platform settings -> security policy -> limiting multiple concurrent sessions for a single user
- **Added** platform settings -> security policy -> limiting the maximum number of concurrent session connections 
  for the system

#### Optimization

- **Optimized** support for customized parameters in Webhook URL

#### Fixes

- **Fixed** issue with errors when filling in the mail server
- **Fixed** issue where LDAP users could not join user groups

## 2023-07-06

### v0.18.1

#### Fixes

- **Fixed** issue where workspace cluster names could be empty
- **Fixed** issue with Folder Admin role permissions in the workspace authorization list
- **Fixed** issue with Ghippo upgrade from 0.17 to 0.18
- **Fixed** issue where options for selecting cluster types were the same when adding shared resources
- **Fixed** issue where mesh-type resources in the resource group list could not be unbound

## 2023-06-29

### v0.18.0

#### Features

> - **Added** move workspace

Reviewer: ```suggestion
- **Added** the feature of workspace migration
```
